### ISSUE TITLE
Is is possible to skip a training sample?

### ISSUE THREAD

**Issue by tankche1** [open-mmlab/mmdetection#2467]
For example, i can make loss = {'loss_cls'=Variable(torch.Tensor([0]).float().cuda())}
However, it seems that the value of loss dict can not all be zeros. Otherwise, it will be stuck.
Any solution to that?

**Comment by hellock** [comment]
The simplest way I can propose is to multiply the normal loss with 0. You cannot skip part of the module when using Pytorch's DistributedDataParallel. For some hacks if you have a good understanding of it, please refer to #2280.
